containers/Ideas/IdeaCreate.styles.js

```diff
@@ -7,7 +7,9 @@ import { SPACINGS } from '../../theme/spacings'
 export const styles = StyleSheet.create({
   container: {
     paddingHorizontal: SPACINGS.multiplyBy(0.75),
-    backgroundColor: COLORS.paper
+  },
+  header: {
+    paddingTop: SPACINGS.multiplyBy(0.25)
```

Comment:
I wonder if this should just be on the header and we unify the top padding on all containers so we don't have it set individually all over the place?